tests/test_pois.py

```diff
@@ -0,0 +1,274 @@
+        self.assertEqual(response['result']['poi']['x_coord'], poi_create_complete['x_coord'])
+        self.assertEqual(response['result']['poi']['y_coord'], poi_create_complete['y_coord'])
+        for i in range(len(response['result']['poi']['links'])):
+            self.assertEqual(response['result']['poi']['links'][i]['link_url'], poi_create_complete['links'][i]['link_url'])
+            self.assertEqual(response['result']['poi']['links'][i]['display_name'], poi_create_complete['links'][i]['display_name'])
+        for j in range(len(response['result']['poi']['media'])):
+            self.assertEqual(response['result']['poi']['media'][j]['content_url'], poi_create_complete['media'][j]['content_url'])
+            self.assertEqual(response['result']['poi']['media'][j]['caption'], poi_create_complete['media'][j]['caption'])
+        response_story_ids = [k['_id'] for k in response['result']['poi']['stories']]
+        self.assertEqual(sorted(response_story_ids), sorted(poi_create_complete['story_ids']))
+        os.environ['POI_ID'] = str(response['result']['poi']['_id'])
+
+    def test0_2_get_poi_by_id(self, poi_id=0):
+        if poi_id == 0:
+            poi_id = int(os.environ.get('POI_ID'))
+        r = requests.get('http://127.0.0.1:5000/pois/{}'.format(poi_id))
+        response = r.json()
+        self.assertEqual(response['code'], 200)
+        self.assertEqual(response['result']['poi']['_id'], poi_id)
+
+        self.assertEqual(response['result']['poi']['name'], poi_create_complete['name'])
+        # self.assertEqual(response['result']['poi']['date']
+        self.assertEqual(response['result']['poi']['description'], poi_create_complete['description'])
+        self.assertEqual(response['result']['poi']['map_year'], poi_create_complete['map_year'])
+        self.assertEqual(response['result']['poi']['x_coord'], poi_create_complete['x_coord'])
+        self.assertEqual(response['result']['poi']['y_coord'], poi_create_complete['y_coord'])
+
+        #Account for time zones (see above commented out line)
+        #Future Testing needed for links, media, stories (would require several loops based on json response)
+
+    def test0_3_get_pois_by_map_year(self):
+        map_year = poi_create_complete['map_year']
+
+        #get request by map_year
+
+        r1 = requests.get('http://127.0.0.1:5000/pois?map_year={}'.format(map_year))
+        response1 = r1.json()
+
+        self.assertEqual(response1['code'], 200)
+
+        for i in response1['result']['pois']:
+            self.test0_2_get_poi_by_id(poi_id = i['_id'])
+            self.assertEqual(i['map_year'], map_year)
+
+    def test0_3_get_pois_by_story_id(self):
+        story_ids = poi_create_complete['story_ids']
+
+        #get request by story_id
+
+        for curr_story_id in story_ids:
+            r2 = requests.get('http://127.0.0.1:5000/pois?story_id={}'.format(curr_story_id))
+            response2 = r2.json()
```

Comment:
since you broke these out, they are the only request/responses in the function, so you don't (and shouldn't) still call them `r2`/`response2` since it might make people wonder where `r1`/`response1` is